package.json

```diff
@@ -14,7 +14,11 @@
 		"node": ">=12"
 	},
 	"scripts": {
-		"test": "xo && nyc ava && markdownlint '**/*.md'",
+		"lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
```

Comment:
I used `npm-run-all` for a long time, I think simple `run-p lint:*` is good enough.

Reply:
I can switch to `run-p` but don't you think `--continue-on-error --aggregate-output` have benefits as mentioned in PR description?

Reply:
I don't have strong opinion, I can give it a shot.

Reply:
Actually `--aggregate-output` causes it to lose the output formatting so I removed that. But switched to `run-p`.